consensus/consensusfsm/consensus_ttl.go

```diff
@@ -11,19 +11,41 @@ import (
 	"github.com/iotexproject/iotex-core/v2/blockchain/genesis"
 )
 
-// DefaultDardanellesUpgradeConfig is the default config for dardanelles upgrade
-var DefaultDardanellesUpgradeConfig = DardanellesUpgrade{
-	UnmatchedEventTTL:            2 * time.Second,
-	UnmatchedEventInterval:       100 * time.Millisecond,
-	AcceptBlockTTL:               2 * time.Second,
-	AcceptProposalEndorsementTTL: time.Second,
-	AcceptLockEndorsementTTL:     time.Second,
-	CommitTTL:                    time.Second,
-	BlockInterval:                5 * time.Second,
-	Delay:                        2 * time.Second,
-}
+var (
+	// DefaultDardanellesUpgradeConfig is the default config for dardanelles upgrade
+	DefaultDardanellesUpgradeConfig = DardanellesUpgrade{
+		UnmatchedEventTTL:            2 * time.Second,
+		UnmatchedEventInterval:       100 * time.Millisecond,
+		AcceptBlockTTL:               2 * time.Second,
+		AcceptProposalEndorsementTTL: time.Second,
+		AcceptLockEndorsementTTL:     time.Second,
+		CommitTTL:                    time.Second,
+		BlockInterval:                5 * time.Second,
+		Delay:                        2 * time.Second,
+	}
+	// DefaultWakeUpgradeConfig is the default config for wake upgrade
+	DefaultWakeUpgradeConfig = WakeUpgrade{
+		UnmatchedEventTTL:            2 * time.Second,
+		UnmatchedEventInterval:       100 * time.Millisecond,
+		AcceptBlockTTL:               1 * time.Second,
+		AcceptProposalEndorsementTTL: time.Second,
+		AcceptLockEndorsementTTL:     500 * time.Millisecond,
+		CommitTTL:                    500 * time.Millisecond,
```

Comment:
`AcceptProposalEndorsement` is identical to `AcceptLockEndorsementTTL` in the eyes of consensus stages, so should give them same amount of time. the below is a more balanced choice?
```
AcceptProposalEndorsementTTL: 700 * time.Millisecond,
AcceptLockEndorsementTTL: 700 * time.Millisecond,
CommitTTL: 600 * time.Millisecond
```

Reply:
Yes, the lock phase is just a network vote and is not computation-intensive.